### PR TITLE
Refactoring and Cleaning Up

### PR DESCRIPTION
**Pull request by MoMMde**
SpongeBobCleaningGIF

### CODE REVIEW

File: src/main/java/de/l4zs/connectfour/ConnectFour.java [hunk 1]
```diff
@@ -40,11 +42,9 @@ public void registerCommands() {
     }
 
     @Override
-    public void onDisable() {
-        // Plugin shutdown logic
-    }
+    public void onDisable() {}
```
Comment (l4zs) [inline src/main/java/de/l4zs/connectfour/ConnectFour.java:45]: useless

File: src/main/java/de/l4zs/connectfour/ConnectFour.java [hunk 2]
```diff
@@ -11,16 +11,18 @@
...
+
     @Override
     public void onEnable() {
         try {
             registerEvents(Bukkit.getPluginManager());
             registerCommands();
 
-            Bukkit.getLogger().info("§4§lSponsored by banko.gay");
-            Bukkit.getLogger().info("§8● §cVersion: " + getDescription().getVersion());
+            getLogger().info("§4§lSponsored by banko.gay");
```
Comment (l4zs) [inline src/main/java/de/l4zs/connectfour/ConnectFour.java:24]: already changed not pushed tho

File: src/main/java/de/l4zs/connectfour/listener/InventoryCloseListener.java [hunk 3]
```diff
@@ -15,7 +15,7 @@ public void onInventoryClose(InventoryCloseEvent event) {
         Player player = (Player) event.getPlayer();
         ConnectFourPlayer cp = ConnectFourPlayer.getConnectFourPlayer(player);
         if (cp.isInGame()) {
-            Bukkit.getScheduler().runTaskLater(ConnectFour.getPlugin(ConnectFour.class), () -> player.openInventory(event.getInventory()), 0);
+            Bukkit.getScheduler().runTaskLater(ConnectFour.getInstance(), () -> player.openInventory(event.getInventory()), 0);
```
Comment (l4zs) [inline src/main/java/de/l4zs/connectfour/listener/InventoryCloseListener.java]: already changed not pushed tho

File: src/main/java/de/l4zs/connectfour/util/Message.java [hunk 4]
```diff
@@ -68,6 +68,6 @@ private Message() {
 
     public enum LanguageType {
         GERMAN,
-        ENGLISH;
+        ENGLISH
```
Comment (l4zs) [inline src/main/java/de/l4zs/connectfour/util/Message.java:71]: bad practice if we would want to add parameters for this enum

File: src/main/java/de/l4zs/connectfour/util/ConnectFourItem.java [hunk 5]
```diff
@@ -30,8 +30,7 @@ private ConnectFourItem() {
         return connectFourItems;
     }
 
-    public static ItemStack getGameLeaveItem() {
-        return new ItemBuilder(Material.DARK_OAK_DOOR, 1).setName("§cSpiel verlassen").build();
-    }
+    public static ItemStack quitItem = new ItemBuilder(Material.DARK_OAK_DOOR, 1).setName("§cSpiel verlassen").build();
```
[the remark this answers is not in the document]
Comment (MoMMde) [reply src/main/java/de/l4zs/connectfour/util/ConnectFourItem.java:33]: Weil man den ItemStack sonst immer neu builded anstatt einmal

[the remark this answers is not in the document]
Comment (l4zs) [reply src/main/java/de/l4zs/connectfour/util/ConnectFourItem.java:33]: well then the itemstacks aren't unique causing many issues

**Pull request closed without merging**